Changelog.md

```diff
@@ -25,7 +26,6 @@ Compiler Features:
  * Yul Optimizer: Add rule to convert ``mod(add(X, Y), A)`` into ``addmod(X, Y, A)``, if ``A`` is a power of two.
  * Yul Optimizer: Add rule to convert ``mod(mul(X, Y), A)`` into ``mulmod(X, Y, A)``, if ``A`` is a power of two.
 
-
```

Comment:
Probably also by accident.

Reply:
Yeah, but it actually came from [22c7cd22](https://github.com/ethereum/solidity/commit/22c7cd22b92d3e0c9d81880492cf8c56ea0370cd), so I think it is on develop branch now...